config/examples/inject-launcher/kustomization.yaml

```diff
@@ -0,0 +1,27 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+# Generate ConfigMap from files
+configMapGenerator:
+  - files:
+      - __init__.py
+      - launcher.py
+    name: launcher-code
+    options:
+      disableNameSuffixHash: true
```

Comment:
If `kustomize` is invoked per inference server, there will be multiple copies of the same ConfigMap --- i.e., with the same name, which cannot be?